pkg/proto/nbcontract/v1/clusterconfig.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+import "pkg/proto/nbcontract/v1/loadbalancerconfig.proto";
+import "pkg/proto/nbcontract/v1/vmtype.proto";
+
+// Cluster Config fields stored in azure.json used by cloud-provider-azure
+message ClusterConfig {
+  string resource_group = 1;
+  string location = 2;
+  VmType vm_type = 3;  
```

Comment:
suggest putting the VmType enum in this proto file as we are doing so for others